README.md

```diff
@@ -38,7 +38,7 @@ react-native link react-native-auth0
 
 #### Android
 
-In the file `android/src/app/AndroidManifest.xml` you must make sure the main activity of the app has launch mode value of `singleTask` and that it has the following intent filter
+In the file `android/app/src/main/AndroidManifest.xml` you must make sure the **MainActivity** of the app has a **launchMode** value of `singleTask` and that it has the following intent filter:
```

Comment:
Path is wrong ![image](https://user-images.githubusercontent.com/3900123/31180443-0455fae2-a8f5-11e7-9bf5-52edae3e9d20.png)

Reply:
That is the path inside the react-native-auth0 module. Not the path for the users project

Reply:
<img width="670" alt="screen shot 2017-10-04 at 15 14 56" src="https://user-images.githubusercontent.com/928115/31180723-447b5f82-a917-11e7-9bec-056f9b7adab7.png">

Reply:
The doc is in the context of configuring the users react-native project. Just as the `ios` one is.